report/renderer/report-ui-features.js

```diff
@@ -60,6 +60,7 @@ export class ReportUIFeatures {
    */
   initFeatures(lhr) {
     this.json = lhr;
+    this._fullPageScreenshot = Util.getFullPageScreenshot(lhr);
```

Comment:
Why do we need to store this on the object? `getFullPageScreenshot` isn't an expensive operation.

Reply:
Typechecking. It needed to be set to a variable somewhere to do type narrowing.